.github/workflows/docs.yml

```diff
@@ -5,6 +5,7 @@ on:
     paths:
       # Something has to change in doc/ for thos workflow to be run.
       - "doc/**"
+      - "etc/schema.json"
```

Comment:
Oh, the docs job should still run on schema updates. There is tooling that generates an eve index from the schema that we would want to have run on schema updates I think.